### DISCUSSION TITLE
Mobheads not Identified since the last updates

### DISCUSSION THREAD

**Discussion by thenullmatrix** [Zrips/CMI#2872] [Q&A]
**Question: **
Not sure what is causing this issue we have ever since the recent changes to mobheads that drop when we use our auction system we get an error There is nothing in you hand.  Any Idea on what changed in the creation of these drops to cause this now?

**Comment by Zrips** [comment]
What error are you getting? And i need more information like atleast the server version.

**Comment by thenullmatrix** [comment]
The only errors I get is There is nothing in your hand or All items must be identical. This is just with one just mobhead. Work fine with all other items and player heads just not the cmi mobhead versions.

CMI ver 8.6.20.0
Server version: Spigot 1.15.2-R0.1-SNAPSHOT

Our auction plugin is Craftbay 2.26

**Comment by thenullmatrix** [comment]
Just a note we have been able to use these mob head drops before with no issue.

**Comment by Zrips** [comment]
Craftbay could be using some special data. Could you provide pictures of items with which you have issues and which works correctly, how their lore looks like and what outout you have with /cmi itemnbt

**Comment by thenullmatrix** [comment]
Here is one I have issue with
<img width="1308" alt="Screen Shot 2020-03-12 at 6 33 38 PM" src="https://user-images.githubusercontent.com/19312899/76572410-5b6a9000-6490-11ea-99d7-5ba1ca870495.png">
and here is one that works (had to do two images)
![Screen Shot 2020-03-12 at 6 34 57 PM](https://user-images.githubusercontent.com/19312899/76572429-7210e700-6490-11ea-8437-62f4a9169433.png)
![Screen Shot 2020-03-12 at 6 35 13 PM](https://user-images.githubusercontent.com/19312899/76572433-75a46e00-6490-11ea-9ffa-cc072e56ed59.png)

I am not sure on lore (don't know how to get this info) The one that works was from another heads source. Any of our old ones from cmi worked when from a chest but if I did /cmi itemnbt or placed and broke the head they stopped working.

I am just looking for help on why this is happening I am no Java expert and welcome any help. Its ok if it takes you some time to reply. We love CMI and understand it takes alot of work to maintain.
